### PR TITLE
Chore: Type Rudderstack and AppInsights window APIs

### PR DESCRIPTION
**Pull request by joshhunt**
**What this PR does / why we need it**:

loosely types the analytics sdks on window 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -382,6 +382,7 @@
     "regenerator-runtime": "0.13.9",
     "reselect": "4.1.6",
     "rst2html": "github:thoward/rst2html#990cb89f2a300cdd9151790be377c4c0840df809",
+    "rudder-sdk-js": "^2.13.0",
```
Comment (ashharrison90) [inline package.json:385]: if we're just using this for types, can this be a `devDep`? 🤔

**Review by ashharrison90 — APPROVED** [review]
looks great! 🙌 

added a couple of questions, they don't need to hold out this PR as this is already an improvement.

File: package.json (hunk 1 above)
Comment (joshhunt) [reply package.json:385]: Yeah good question. I didn't exactly know the best place to put it, but i think dev deps might be better? Either way, it's not like we actually run/use a 'production' node_modules anyway.

File: public/app/core/services/echo/backends/analytics/RudderstackBackend.ts [hunk 2]
```diff
@@ -32,7 +48,7 @@ export class RudderstackBackend implements EchoBackend<PageviewEchoEvent, Rudder
       cache: true,
     });
 
-    const rds = ((window as any).rudderanalytics = []);
+    const tempRudderstack = ((window as any).rudderanalytics = []);
```
[the remark this answers is not in the document]
Comment (ashharrison90) [reply public/app/core/services/echo/backends/analytics/RudderstackBackend.ts:51]: ahhhhhhhhh got it, ok makes sense, thanks! 🙌

File: package.json (hunk 1 above)
Comment (ashharrison90) [reply package.json:385]: yep true, it doesn't really matter either way 🙂

**Pull request merged**